cmp/form/dropdown/BaseComboField.js

```diff
@@ -31,20 +36,18 @@ export class BaseComboField extends BaseDropdownField {
 
     forceSelectionToOptionOrRevert() {
         const {internalOptions, internalValue} = this;
+        if (!internalValue) return;
```

Comment:
doCommit will fire onBlur and call this method. If this happens before an internalValue has been set e.g. when choosing an option from the dropdown this method would throw.